lib/rubocop/cop/mixin/negative_conditional.rb

```diff
@@ -9,10 +9,13 @@ module NegativeConditional
       include IfNode
 
       def_node_matcher :single_negative?, '(send !(send _ :!) :!)'
+      def_node_matcher :empty_brace?, '(begin)'
```

Comment:
It might also be overkill to use a node matcher for this. What do you think about something like (not tested):

```
def empty_condition?(condition)
  node.begin_type? && node.children.emtpy?
end
```

One of the downsides of `def_node_matcher` is that it will circumvent auto-complete in your editor, which makes these methods pretty undiscoverable. 😞